docs/cvl/cvl2/migration.md

```diff
@@ -2,9 +2,29 @@ Migration guide
 ===============
 
 This section gives a step-by-step process for migrating your specs from CVL 1 to
-CVL 2.
+CVL 2.  It only addresses the changes that are most likely to arise; for full
+details see {doc}`changes`.
 
-## Step 1: Familiarize yourself with the CVL 2 changes
+Here is an outline of the migration process:
+
+```{contents}
+:local:
+:depth: 1
+```
+
+If you have any questions, please ask for help!
+
+```{todo}
+How do you ask for help?
+```
+
+## Step 0: Install CVL 2
+
+```{todo}
+How do you get CVL 2?
```

Comment:
`pip install certora-cli-beta`

Reply:
We should also include instructions on using both simultaneously and downgrading to LTS version if necessary.